temporal-sdk/src/main/java/io/temporal/common/interceptors/WorkflowOutboundCallsInterceptorBase.java

```diff
@@ -156,6 +156,11 @@ public void upsertTypedSearchAttributes(SearchAttributeUpdate<?>... searchAttrib
     next.upsertTypedSearchAttributes(searchAttributeUpdates);
   }
 
+  @Override
+  public void upsertMemo(Map<String, Object> memo) {
```

Comment:
Does Java offer the ability for me to see the whole memo collection inside the workflow? For instance if I wanted to know all memo keys in a workflow, can I? Do we need to add that?

Reply:
No and No, at least no one has asked for it and I see no reason this PR would need to add it.

Reply:
Works for me. My concern is if/when we do we need to make sure to more explicitly make sure null means remove from a `Map` POV.